client/services/code_preprocessors/PythonCodePreprocessorService.js

```diff
@@ -54,43 +93,66 @@ tie.factory('PythonCodePreprocessorService', [
       correctnessTests.forEach(function(test, index) {
         testCode += '\n' + [
           VARNAME_TEST_RESULTS + '.append(',
-          '    run_test(' + jsonVariableToPython(test.getInput()) + '))'
+          '    run_correctness_test(' + jsonVariableToPython(test.getInput()) + '))'
         ].join('\n');
       });
 
       return testCode;
-    }
+    };
 
-    return {
-      // Wraps a WRAPPER_CLASS_NAME class around the series of functions in a
-      // given code snippet.
-      _wrapCodeIntoClass: function(code) {
```

Comment:
I'm not removing _wrapCodeIntoClass, I thought -- isn't it further up in the file?